charts/k8s-monitoring/README.md

```diff
@@ -347,9 +352,7 @@ The Prometheus and Loki services may be hosted on the same cluster, or remotely
 | test.nodeSelector | object | `{"kubernetes.io/os":"linux"}` | nodeSelector to apply to the test job. |
 | test.tolerations | list | `[]` | Tolerations to apply to the test job. |
 | traces.enabled | bool | `false` | Receive and forward traces. |
-| traces.processors.batch.maxSize | int | `0` | The upper limit of the amount of data contained in a single batch, in bytes. When set to 0, batches can be any size. |
-| traces.processors.batch.size | int | `16384` | What batch size to use, in bytes |
-| traces.processors.batch.timeout | string | `"2s"` | How long before sending |
+| traces.filters | object | `{"span":[],"spanevent":[]}` | Apply a filter to traces received via the OTLP or OTLP HTTP receivers. See https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.processor.filter/ for details. |
```

Comment:
I was debating that. I might change it just for consistency.
The reason I didn't was only because all traces come in via receivers, so it felt redundant.